source/Stargate-Application-Info-Pharo/PharoOperatingSystemInformationProvider.class.st

```diff
@@ -2,25 +2,25 @@
 I'm a provider of information about the Operating System running this application
 "
 Class {
-	#name : #OperatingSystemInformationProvider,
+	#name : #PharoOperatingSystemInformationProvider,
```

Comment:
Shouldn´t this be UNIXOperatingSystemInformationProvider?

Reply:
This provider works for Windows too, it only shows less detailed information.

Reply:
I think Juan has a good point. When thinking about an OS we think about Linux, Windows, etc. Maybe a better name is "platform"? I asked Microsoft AI "What is the difference between platform and operating system?" and got:

> In the context of computer systems, the terms **platform** and **operating system** have distinct meanings¹²³.
> 
> An **operating system** (OS) is a software that manages a computer's hardware and software resources and provides common services for computer programs². It acts as an intermediary between applications and hardware, managing how applications access hardware and software resources³. An OS provides a set of interconnected tools such as file systems, interprocess communication, process management, network access, human interface, and shells¹.
> 
> On the other hand, a **platform** is a combination of hardware and software that allows software to run². It is a group of technologies used as a base upon which other applications, processes, or technologies are developed³. For instance, Windows and macOS are two platforms or operating systems¹. When it comes to different versions of Windows like Windows XP and Windows 7, they can be considered different versions of the same platform¹.
> 
> To summarize, an operating system is the software that manages a computer's resources and provides common services for computer programs. A platform is the hardware and software combination that allows software to run².
> 
> Source: Conversation with Bing, 9/28/2023
> (1) Platform vs operating system - Super User. https://superuser.com/questions/514459/platform-vs-operating-system.
> (2) What is the Difference Between Operating System and Platform?. https://similardifferent.com/difference-between-operating-system-and-platform/.
> (3) What is difference between platform and operating system?. https://www.compuhoy.com/what-is-difference-between-platform-and-operating-system/.
> (4) Quick Answer: What is difference between platform and operating system?. https://frameboxxindore.com/other/quick-answer-what-is-difference-between-platform-and-operating-system.html.